cmd/operator/main.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 var (
-	log = logger.NewDTLogger()
+	log = logger.NewDTLogger().WithName("main")
```

Comment:
in `ctrl.SetLogger(...)` a new logger without name is used. Could this one be reused there?